isb_web/main.py

```diff
@@ -100,8 +100,6 @@
 
 @app.on_event("startup")
 def on_startup():
-    # method to initialize all of the available models in config
-    MetadataModelLoader.initialize_models()
```

Comment:
remove this and instead load the model by calling `get_xxx_model()` whenever we need to use it.